synapseclient/client.py

```diff
@@ -658,6 +660,31 @@ def _print_transfer_progress(self, *args, **kwargs):
         if self.silent is not True:
             cumulative_transfer_progress.printTransferProgress(*args, **kwargs)
 
+    ############################################################
+    #                      Service methods                     #
+    ############################################################
+
+    _services = {
+        "json_schema": "JsonSchemaService",
+    }
+
+    def get_available_services(self):
+        services = self._services.keys()
+        return list(services)
+
+    def service(self, service_name: str):
+        import synapseclient.services
```

Comment:
Is there a reason for this import to be here and not part of the global imports?

Reply:
To avoid circular imports. When I revisit this PR, I hope to revisit the import order and [method](https://stackoverflow.com/a/37126790) in the hope of moving this import statement to the top.